expression/schema.go

```diff
@@ -34,9 +34,10 @@ func (ki KeyInfo) Clone() KeyInfo {
 
 // Schema stands for the row schema and unique key information get from input.
 type Schema struct {
-	Columns   []*Column
-	Keys      []KeyInfo
-	MaxOneRow bool
+	Columns      []*Column
+	Keys         []KeyInfo
+	TblID2handle map[int64][]*Column
```

Comment:
The value type of the map is the handle columns not normal columns, i may change it to `TableID2HandleCols`